README.md

```diff
@@ -385,6 +385,54 @@ command.
 Other options like `--wait-completion` are also available and work in the same way as for the `cloudos job run` command.
 Check `cloudos bash job --help` for more details.
 
+#### Send a bash array-job to CloudOS (parallel sample processing)
+
+The subcommand `bash array-job` can list the columns of an array file by using the option `--list-columns` when adding `--array-file <CloudOS File Explorer Path>`. The columns will be listed one per line, example:
+
+```console
+Columns:
+    - column1
+    - column2
+    - column3
+```
+
+When running a bash array job, the following options are available to customize the behavior:
+
+##### Array File
+- **`--array-file`**: Specifies the path to a file containing a set of columns useful in running the bash job. This option is **required** when using the command `bash array-job`.
+
+##### Separator
+- **`--separator`**: Defines the separator to use in the array file. Supported separators include:
+    - `,` (comma)
+    - `;` (semicolon)
+    - `tab`
+    - `space`
+    - `|` (pipe)
+This option is only applicable when `--array-file` is provided and is **required**.
+
+##### List Columns
+- **`--list-columns`**: Lists the columns available in the array file. This is useful for inspecting the structure of the file. This flag disables sending the job, it just prints the column list.
+
+##### Array File Project
+- **`--array-file-project`**: Specifies the name of the project to use when running the array file, if it is different from the project specified by `--project-name`.
+
+##### Disable Column Check
+- **`--disable-column-check`**: Disables the validation of columns in the array file. 
```

Comment:
Could you explain what it implies? We need to explain what validations we do and so the user can understand what is disabled.